docs/fundamentals/client/client.md

```diff
@@ -668,7 +663,26 @@ with Flow().add() as f, open('output.txt', 'w') as fp:
     )
 ```
 
+````{admonition} What errors can be handled by the callback
+:class: caution
+Callback can handle errors that are caused by Executors raising an Exception.
+It will not receive exceptions or from the Gateway having connectivity errors with the Executors or between the Client and the Gateway.
+````
 
+### Continue streaming when an error occurs
+
+`client.post()` accepts `continue_on_error` parameter. When set to True, `Client` will keep trying to send the following requests. `continue_on_error` parameter will only apply
+to Exceptions caused by Executor exceptions, but in case of network connectivity issues, an Exception will be raised.
```

Comment:
```suggestion
`client.post()` accepts a `continue_on_error` parameter. When set to `True`, the Client will keep trying to send the remaining requests. The `continue_on_error` parameter will only apply
to Exceptions caused by an Executor, but in case of network connectivity issues, an Exception will be raised.
```